src/forms/datacenterForm.tsx

```diff
@@ -1,142 +1,140 @@
-                    errors.push(err.response.data.failure_message as string);
-                    this.setState({
-                        errors: errors
-                    });
-                });
-            }
-        }
-    };
+      }
 
-    handleChange = (field: { [key: string]: any }) => {
-        this.setState({
-            values: updateObject(this.state.values, {
-                ...field
-            })
+      const resp = this.props.submitForm(this.state.values, this.headers);
+      if (resp) {
+        resp.catch(err => {
+          console.log(err.response.data.failure_message);
+          let errors: Array<string> = this.state.errors;
+          errors.push(err.response.data.failure_message as string);
+          this.setState({
+            errors: errors
+          });
         });
-    };
-    selectText = (event: any) => event.target.select();
+      }
+    }
+  };
 
-    render() {
-        const { values } = this.state;
-        return (
-            <div className={Classes.DARK + " login-container"}>
-                {this.state.errors.map((err: string) => {
-                    return <Callout intent={Intent.DANGER}>{err}</Callout>;
-                })}
-                <form
-                    onSubmit={this.handleSubmit}
-                    className="create-form bp3-form-group"
-                >
-                    <FormGroup label="Name (required)" inline={false}>
-                        <Field
-                            field="name"
-                            placeholder="name"
-                            value={values.name}
-                            onChange={this.handleChange}
-                        />
-                    </FormGroup>
-                    <FormGroup label="Abbreviation (required - up to 6 characters)" inline={false}>
-                        <Field
-                            placeholder="abbreviation"
-                            onChange={this.handleChange}
-                            value={values.abbreviation}
-                            field="abbreviation"
-                        />
-                    </FormGroup>
+  handleChange = (field: { [key: string]: any }) => {
+    this.setState({
+      values: updateObject(this.state.values, {
+        ...field
+      })
+    });
+  };
+  selectText = (event: any) => event.target.select();
 
-                    <Button className="login-button" type="submit">
-                        Submit
-            </Button>
-                </form>
-            </div>
-        );
-    }
+  render() {
+    const { values } = this.state;
+    return (
+      <div className={Classes.DARK + " login-container"}>
+        {this.state.errors.map((err: string) => {
+          return <Callout intent={Intent.DANGER}>{err}</Callout>;
+        })}
+        <form
+          onSubmit={this.handleSubmit}
+          className="create-form bp3-form-group"
+        >
+          <FormGroup label="Name (*)" inline={false}>
```

Comment:
for consistency with the other forms, this would be "Name*" and "Abbreviation* (up to 6 characters)" I know that's picky but this was a picky change to begin with muahaa

Reply:
Done!